lib/cisco_node_utils/vxlan_vtep_vni.rb

```diff
@@ -29,8 +29,9 @@ class VxlanVtepVni < NodeUtil
     def initialize(name, vni, assoc_vrf=false, instantiate=true)
       @name = name
       @vni = vni
-      @assoc_vrf = assoc_vrf
+      @assoc_vrf = assoc_vrf == :true ? true : false
```

Comment:
This won't work if `assoc_vrf` is a `TrueClass` object.  While I understand that puppet is passing this as a `Symbol` other orchestration frameworks that use this may not pass it as a symbol.  I would prefer that we leave this as `@assoc_vrf = assoc_vrf` and just make sure to pass the appropriate thing in puppet.  (https://github.com/cisco/cisco-network-puppet-module/pull/142/files#diff-1323926772cf8643b976e839a2d0f0fcR167)

```
assoc_vrf = @resource[:assoc_vrf] == :true ? true : false
@vtep_vni = Cisco::VxlanVtepVni.new(@resource[:interface], @resource[:vni], assoc_vrf)
```